frontend/packages/topology/src/components/application-panel/TopologyApplicationResources.tsx

```diff
@@ -18,7 +18,9 @@ const TopologyApplicationResources: React.FC<TopologyApplicationResourcesProps>
 }) => {
   const { t } = useTranslation();
   const resourcesData = resources.reduce((acc, { resource }) => {
-    acc[resource.kind] = [...(acc[resource.kind] ? acc[resource.kind] : []), resource];
+    if (resource?.kind) {
+      acc[resource.kind] = [...(acc[resource.kind] ? acc[resource.kind] : []), resource];
+    }
```

Comment:
nice to have this check to avoid NPE, but with this change Helm release showing in the sidepanel for the group is not covered right?

Reply:
No, that we will do as part of another ticket.

Reply:
@invincibleJai @vikram-raj Where can I find the follow-up ticket for this?